File: Java/src/ToLowerCaseTest.java
import org.junit.Assert;
import org.junit.Test;

public class ToLowerCaseTest {

    @Test
    public void testToLowerCase() {
        ToLowerCase tlc = new ToLowerCase();

        Assert.assertEquals(tlc.toLowerCase("Hello"), "hello");
        Assert.assertEquals(tlc.toLowerCase("here"), "here");
        Assert.assertEquals(tlc.toLowerCase("LOVELY"), "lovely");
    }

}

File: Kotlin/src/ToLowerCaseTest.kt
import org.junit.Assert
import org.junit.Test

class ToLowerCaseTest {

    @Test
    fun testToLowerCase() {
        val tlc = ToLowerCase()

        Assert.assertEquals(tlc.toLowerCase("Hello"), "hello")
        Assert.assertEquals(tlc.toLowerCase("here"), "here")
        Assert.assertEquals(tlc.toLowerCase("LOVELY"), "lovely")
    }

}
